### ISSUE TITLE
Issue with full inventories. 

### ISSUE THREAD

**Issue by codebycam** [diecode/KillerMoney#29]
**Server:** Paper 1.13.2 Build 588
**KillerMoney:** 4.3.2

I've checked over the configuration files and can't see anything for this. Players can't pick up the money if their inventory is full due to the money being an "item":

```
            Items:
                '1':
                    ItemStack:
                        ==: org.bukkit.inventory.ItemStack
                        type: GOLD_NUGGET
                    Random-amount: 1 ? 5
                    Chance: 100%
```

Cheers

**Comment by milothedalmador** [comment]
Having the same issue. Any ideas when/if it'll be sorted?

**Comment by kgahor** [comment]
The PlayerPickupItemEvent only call, when player has empty inventory slot. I can solve this with protocollib, but I don't want this dependency. I am thinking, what should I do.

**Comment by codebycam** [comment]
Cheers for looking into it and confirming the bug. 
Could this be fixed when the plugin is updated to 1.14? I don't believe having it rely on ProtocolLib would necessarily be a bad thing, as many other plugins out there already depend on it.

**Comment by MesterDavid** [comment]
Same problem.
Aaaand half a year passed and nothing happened 😢
[Last Spigot 1.12.2, last KM]
